<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="NameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AuthorLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RatingLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="YearLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PriceLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GenrelineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100000</width>
     <height>100000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
